Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityOptimizeWindow.cs

```diff
@@ -0,0 +1,607 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities.Editor;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Editor
+{
+    /// <summary>
+    /// Build window - Utility for developers to automate optimization of their Unity scene/project
+    /// </summary>
+    public class MixedRealityOptimizeWindow : EditorWindow
+    {
+        private int selectedToolbarIndex = 0;
+        private readonly string[] ToolbarTitles = { "Project Optimizations", "Scene Optimizations", "Shader Optimizations" };
+        private enum ToolbarSection
+        {
+            Project = 0,
+            Scene,
+            Shader
+        };
+
+        private Vector2 windowScrollPosition = new Vector2();
+        private bool showProjectOptimizations = true;
+        private bool showSceneOptimizations = true;
+        private bool showShaderOptimizations = true;
```

Comment:
Getting a warning in 2019.1.1,  `The field 'MixedRealityOptimizeWindow.showShaderOptimizations' is assigned but its value is never used`